internal/pkg/bootstrap/bootstrap.go

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
```

Comment:
This file name is an example of "stutter". How can it be named differently?

Reply:
I wouldn't name it differently.  

The overall purpose of the package is to provide a bootstrap mechanism.  The public implementation is wholly contained with the bootstrap.go file.  That the implementation file shares the package name isn't an issue for me as the file isn't referenced by name anywhere.

I purposefully named the function entry point something other than "Bootstrap()" -- e.g. "Run()" -- so that we didn't end up with stutter in the code (i.e. the code reads bootstrap.Run() and not bootstrap.Bootstrap()).

Reply:
I see your point from a usage perspective. From a project perspective though we've tried to be consistent in also not having package/package.go.

Reply:
No disrespect intended -- that seems like an arbitrary rule to me.  The Go blog defines stutter as:

> Avoid stutter. Since client code uses the package name as a prefix when referring to the package contents, the names for those contents need not repeat the package name. The HTTP server provided by the http package is called Server, not HTTPServer. Client code refers to this type as http.Server, so there is no ambiguity.

By that definition, this isn't stutter.  

I believe bootstrap.go is the appropriate name. 

I don't have an alternative naming suggestion.  Tell me what you'd like me to rename it as and I'll do so.